File: CS1B/even_odd.py
def main():

	print ("--------------------------------------------------");
	print ("A small program to determine whether the input number is even or odd");
	print("----------------------------------------------------");
	print;
	
	num = int(raw_input("Enter a number: "));
	
	if num == 0:
		print "%d is zero!" % (num);
	elif num%2 == 0:
		print "%d is even!" % (num);
	elif num%2 != 0:
		print "%d is odd!" % (num);
	else:
		print "Whoa!";
		
	close = raw_input();
		

	
	
if __name__ == "__main__":
	main()

File: CS1B/even_odd.c
#include<stdio.h>
#include<conio.h>
void main(void)
{
	int a;
	printf("\nA PROGRAM TO FIND OUT WHETHER THE INPUT NUMBER IS EVEN OR ODD");
	printf("\n-------------------------------------------------------------");
	printf("\n\nEnter a number: ");
	scanf("%d", &a);
	if(a%2 == 0)
		printf("You've entered an even number!");
	else
		printf("You've entered a odd number!");
	getch();
}
